src/ploomber/executors/parallel.py

```diff
@@ -90,6 +89,30 @@ class Parallel(Executor):
     >>> dag = DAG(executor=Parallel(processes=2)) # customize
 
 
+    DAG can exit gracefully on function tasks (PythonCallable):
+
+    >>> # A PythonCallable function that raises DAGBuildEarlyStop
+    >>> def early_stop_root(product):
+    >>>     raise DAGBuildEarlyStop('Ending gracefully')
+
+    >>> # Since DAGBuildEarlyStop is raised, DAG will exit gracefully.
+    >>> dag = DAG(executor='parallel') # use with default values
+    >>> PythonCallable(early_stop_root, File('file.txt'), dag)
+    >>> dag.build()  # This will return None
+
+
+    DAG can also exit gracefully on notebook tasks:
+
+    >>> # A function that calls on a task-level function fn
+    >>> def touch_root(fn):
+    >>>     Path(str(fn)).touch()
+
+    >>> # Use the task-level hook "on_finish" to exit DAG gracefully.
+    >>> dag = DAG(executor='parallel') # use with default values
+    >>> t = PythonCallable(touch_root, File('file.txt'), dag)
```

Comment:
The example should use a NotebookRunner task, instead of a PythonCallable. See an example here: https://github.com/ploomber/ploomber/blob/cd16d6f7b570015995743fce1c602feeb57b4618/src/ploomber/tasks/notebook.py#L570

Reply:
Should this also be NotebookRunner then? https://github.com/ploomber/ploomber/blob/4643e0f69e058c5ed0c26f16b0486386958a6d47/tests/dag/test_dag.py#L983

If it should be NotebookRunner instead of PythonCallable, should I raise an issue?